flutter/news/lib/core/language_extensions.dart

```diff
@@ -0,0 +1 @@
+doNothing({required String because}) => {};
```

Comment:
A convenient method, the idea is to avoid `if` clauses without a proper `else`. 

Usage can be like

```
if (user.authenticated()) {
    user.grantWritePermission();
} else {
   doNothing(because: "We are not allowing unauthenticated users to have edit permissions as per security team recommendations");
}  
```

This helps on documenting when we intentionally do nothing on the else block